src/components/Core.js

```diff
@@ -159,7 +171,17 @@ const Core = (props) => {
 
   return (
     <CoreBackdrop className="core__auth">
-      <CoreBox className="classic">
+      <CoreBox
```

Comment:
Can we move these styles out to App.scss?

Reply:
done